README.md

```diff
@@ -1,3 +1,41 @@
-# Offcial Ziptastic Java Library
-
-## Coming soon!
+# **Offcial Ziptastic Java Library**

```

Comment:
Nice! So I was thinking, and what if we put something like **Offcial Ziptastic Java Library (BETA)** until we get this under test? That way we could soft release it(merge) and then continue working.